### DISCUSSION TITLE
Alternative ingredient units

### DISCUSSION THREAD

**Discussion by llasse** [TandoorRecipes/recipes#3225] [Ideas]
When cooking for a lot of people but even on a smaller scale it would come in handy if we could specify an alternative unit for an ingredient unit, e.g.
2 onions = 100g onions

This could be done on an ingredient base (some ingredients will most always weigh the same, but it'd probably be more practical on a per-recipe base (e.g. some recipes call for a large onion that might already weigh 100g.. )

The work around for me right is to have recipes that are optimized for cooking on larger scale (usually having exact metric measurements or typical units that are used when buying larger amounts) and having smaller scale recipes (where it is much more conventient to just read "2 onions" and not have to look up or take out a scale to weigh 100g of onions..)

**Comment by smilerz** [comment]
you can do this - but there must be a unit first.  in your example it could be:
2 whole onions
2 piece onions
2 medium onions
etc

**Reply by llasse** [reply]
I don’t quite understand what you mean. In my onion example I meant to specifiy the weight of the onions and also providing a “number of pieces” - are you suggesting to just add a separate entry, so that there would be (in my example) one entry with 2 (medium) onions and 100 (g) onions 
?

**Reply by smilerz** [reply]
You would specify the base unit of onion to be gram.  Add a conversion for whole = 100g. 

Recipes would convert the number of whole onions to grams.
